frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -90,7 +114,9 @@ const isIdirDomainSelected = () => {
 
 function userDomainChange() {
     resetVerifiedUserIdentity();
```

Comment:
um..... so we called this "resetVerifiedUserIdentity" here, but the userIdChange method also call the "resetVerifiedUserIdentity" method, so we don't need to call it here anymore, let's clean this up.

Reply:
That's true, thanks for noticing!